src/Admin/BlockAdmin.php

```diff
@@ -81,7 +81,12 @@ public function getObject($id)
             $service = $this->blockManager->get($subject);
 
             $resolver = new OptionsResolver();
-            $service->setDefaultSettings($resolver);
+
+            if (method_exists($service, 'configureSettings')) {
```

Comment:
Do we need this if?

Reply:
Yes, because the method doesn't exist in the stable interface